R/bump-version.R

```diff
@@ -33,7 +34,7 @@ bump_version_impl <- function(which,
   #' 1. Depending on the `which` argument:
   if (which == "dev") {
     #'     - If `"dev"`, [finalize_version()] with `push = FALSE`
-    finalize_version_impl(push = FALSE)
+    finalize_version_impl(push = FALSE, suggest_finalize = edit)
```

Comment:
I find this argument name unclear. edit is clearer. could we rename suggest_finalize to edit?

Reply:
Does the argument name make more sense in the context of `finalize_version_impl()` ?